apps/captain_fact/test/captain_fact/videos/videos_test.exs

```diff
@@ -27,6 +27,15 @@ defmodule CaptainFact.VideosTest do
       assert video.title == "__TEST-TITLE__"
     end
 
+    test "fetch captions" do
```

Comment:
This test should not be placed in `describe "Add video"` bloc. You can get it out of this bloc or create a new `describe "Fetch captions"` bloc to regroup all caption fetch test.

Reply:
Tip: You can run the tests for this specific domain with `mix test test/captain_fact/videos`